### PR TITLE
feat: bump rust to 1.67

### LINKED ISSUE #1012: Update Rust to 1.67
Similar to #930, but with the Rust 1.67 release yesterday.

### PR DESCRIPTION
**Pull request by c3potheds**
## Description
This triggers a rebuild in the Rust Docker image so that rustup can update rust to 1.67, which was released on January 26, 2023.

## Related Issue(s)
Fixes #1012

## How to test

Create a Rust gitpod workspace and run `rustup --version` ¯\\_(ツ)_/¯

## Release Notes

```release-note
update Rust to 1.67
```

## Documentation

https://github.com/gitpod-io/website/issues/3322

### CODE REVIEW

**Review by gengjiawen — APPROVED** [review]

**Comment by kylos101** [comment]
Running the tests, pending them passing, we can approve and merge this PR

cc: @gitpod-io/engineering-workspace

**Comment by gengjiawen** [comment]
blocked by https://github.com/gitpod-io/workspace-images/issues/1016

**Comment by kylos101** [comment]
:wave @c3potheds hey there, I am going to close this PR in favor of https://github.com/gitpod-io/workspace-images/pull/1017/files#diff-fff8b4a42888ae8e237082e153a29023a929b7fd4a5714b1096cc97a97a1bb20R7, and will try to land it in main this week. This will produce a timestamp tagged image that you can use this week, [before the Jammy image lands on Feb 28.](https://www.gitpod.io/changelog#workspace-images-os-and-node-versions-update)

**Pull request closed without merging**
